cypress/e2e/tests/upgrade/after_upgrade.test.ts

```diff
@@ -120,17 +120,20 @@ describe(["@post-upgrade"], "Performing post-upgrade validations", () => {
         exists(binaryApplicationName);
         exists(uploadBinaryApplicationName);
 
-        sourceApplication.analyze();
-        sourceApplication.verifyAnalysisStatus("Completed");
+        uploadBinaryApplication.analyze();
+        uploadBinaryApplication.verifyAnalysisStatus("Completed");
+        uploadBinaryApplication.selectApplication();
 
         binaryApplication.analyze();
         binaryApplication.verifyAnalysisStatus("Completed");
+        binaryApplication.selectApplication();
 
-        uploadBinaryApplication.analyze();
-        uploadBinaryApplication.verifyAnalysisStatus("Completed");
+        sourceApplication.analyze();
+        sourceApplication.verifyAnalysisStatus("Completed");
+        sourceApplication.selectApplication();
     });
 
-    it("Enabling RWX, validating it works, disabling it", function () {
+    it.skip("Enabling RWX, validating it works, disabling it", function () {
```

Comment:
@ibragins here skip is intentional?

Reply:
no, thanks for noticing. Forgot to remove it after the debug.